modules/collector/collector.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2021 Terminus, Inc.
+
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later (AGPL), as published by the Free Software Foundation.
+
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package collector
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/erda-project/erda-infra/base/logs"
+	"github.com/erda-project/erda-infra/base/servicehub"
+	"github.com/erda-project/erda-infra/pkg/parallel-writer"
+	"github.com/erda-project/erda-infra/providers/httpserver"
+	"github.com/erda-project/erda-infra/providers/httpserver/interceptors"
+	"github.com/erda-project/erda-infra/providers/kafka"
+	"github.com/labstack/echo"
```

Comment:
Please distinguish github.com/erda-project/xxx and others.

The order it should be like this:
- standard library
- third-party library
- erda-project related library